docs/_user_guide/best_practices_migrating_from_jupyter.md

```diff
@@ -11,9 +18,9 @@ updateDate: 2023-12-18
 
 # Migrating Processes From JupyterHub to Yen-Slurm
 
-JupyterHub and the interactive Yens are a great resource for developing and debugging code, but is not intended to be final stop for your research computing needs.  If your process requires more resources than the technical limits of JupyterHub and Yen1-5, migrating your process to the <a href="/yen/scheduler.html">`yen-slurm` scheduler</a> will allow you to access more resources.
+JupyterHub and the interactive Yens are a great resource for developing and debugging code, but is not intended to be final stop for your research computing needs.  If your process requires more resources than the technical limits of JupyterHub and Yen1-5, migrating your process to the [`yen-slurm`](/_user_guide/slurm/) scheduler</a> will allow you to access more resources.
 
-# How do I know if it is time to migrate my code from JupyterHub?
+## How do I know if it is time to migrate my code from JupyterHub?
 
 There are three common reasons to migrate your code from JupyterHub:
```

Comment:
3 hour idle..code can be running longer. I think reason #1 is misleading

Reply:
took out 3 hour sentence